.gitignore

```diff
@@ -35,9 +35,10 @@ GettingStarted.html
 
 # mbed build system
 mbed-os/
+mbed-os-ble-utils/
 mbed_settings.py
 mbed_config.h
 *.pyc
 BUILD/
 cmake_build/
-
+.mbedls*
```

Comment:
what is this file?

Reply:
mbedls mock files :-)
https://github.com/ARMmbed/mbed-os-tools/blob/master/packages/mbed-ls/README.md#mocking-renaming-platforms